github3/gists/file.py

```diff
@@ -31,7 +33,8 @@ def _update_attributes(self, attributes):
         #: The size of the file.
         self.size = attributes.get('size')
         #: The content of the file.
-        self.content = attributes.get('content')
+        r = requests.get(self.raw_url)
```

Comment:
So a couple things:

1. Please look through the rest of the code base. We never use `requests.get` (or any other `requests.<method>` function.

2. Anything requiring an extra API call than what the user expected needs to be a separate method

3. That method should return bytes, e.g., `Response.content` not `Response.text`.